tests/test_matrix_utils_invite.py

```
@@ -382,3 +377,95 @@ async def test_on_invite_with_id_in_mapping(mock_logger: MagicMock) -> None:
+    mmrelay.matrix_utils.bot_user_id = "@bot:matrix.org"
+    mmrelay.matrix_utils.matrix_rooms = [
+        {"id": "#mapped:matrix.org", "meshtastic_channel": 0}
+    ]
+
+    await on_invite(mock_room, mock_event)
+    mock_client.join.assert_called_once_with("!unmapped:matrix.org")
+
+
+@pytest.mark.usefixtures("reset_matrix_utils_globals")
+@patch("mmrelay.matrix_utils.logger")
+async def test_on_invite_aliases_list_match(mock_logger: MagicMock) -> None:
```

Ruff (0.15.9)

[warning] Unused function argument: `mock_logger`

(ARG001)